frontend/www/js/omegaup/components/common/Navbar.test.ts

```diff
@@ -63,6 +63,14 @@ describe('Navbar.vue', () => {
     expect(wrapper.find('.navbar-right').text()).toBe(T.navLogIn);
   });
 
+  it('Should handle common navbar to logged user', () => {
+    const wrapper = mount(common_Navbar, {
```

Comment:
```suggestion
    const wrapper = shallowMount(common_Navbar, {
```

`shallowMount` es más ligero que `mount`, ya que no renderiza los componentes hijos del componente actual. Prueba si corriendo `shallowMount` aquí basta, sino lo dejas con el `mount`.